ccd-claim-migration/src/main/java/uk/gov/hmcts/cmc/ccd/migration/idam/api/IdamApi.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.hmcts.cmc.ccd.migration.idam.api;
+
+import org.springframework.cloud.netflix.feign.FeignClient;
+import org.springframework.context.annotation.Bean;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.springframework.context.annotation.Bean'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.reform.hmcts.net/coding_rules#rule_key=squid%3AUselessImportCheck)

Reply:
ok